<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>birdsnest</class>
 <widget class="QWidget" name="birdsnest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>birdsnest</string>
  </property>
  <widget class="QSlider" name="middlez1">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>43</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="middlez2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>108</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd1">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>8</y>
     <width>92</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd2">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>69</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
